packages/@aws-cdk/aws-rds/test/test.parameter-group.ts

```diff
@@ -1,10 +1,10 @@
-import { expect, haveResource } from '@aws-cdk/assert';
+import { countResources, expect, haveResource } from '@aws-cdk/assert';
 import * as cdk from '@aws-cdk/core';
 import { Test } from 'nodeunit';
-import { ClusterParameterGroup, ParameterGroup } from '../lib';
+import { ParameterGroup } from '../lib';
 
 export = {
-  'create a parameter group'(test: Test) {
+  "does not create a parameter group if it wasn't bound to a cluster or instance"(test: Test) {
```

Comment:
nit: why not escape the `'` or reword to say `was not...`

Reply:
I just like it better this way 🙂. Escaping looks ugly, and without contractions the test description reads a little stiff.

But if you're very passionate about it, I can change it 🙂.

Reply:
i'm not :) just what i'd do 😅